portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -936,16 +908,14 @@ definition {
 		}
 
 		task ("Get authorization code") {
-			var authorizationCode = OAuth2.getAuthorizationCode(portalURL = "https://localhost:8443");
+			var authorizationCode = OAuth2.getAuthorizationCode(portalURL = "http://localhost:3000/liferay-oauth2");
```

Comment:
The portalURL should be **https://localhost:8080**. It's the same issue with line 1081.